packages/core/src/components/essentials/tooltip.ts

```diff
@@ -110,6 +125,10 @@ export class Tooltip extends Component {
 
 		// remove the listener on chart-mouseout
 		this.services.events.removeEventListener(Events.Chart.MOUSEOUT, this.handleHideTooltip)
+
+		// remote the listener of the toolbar
```

Comment:
Kindly correct the spelling for remove.

Reply:
Thank you @RiyaJethwa.
Append commit 351a4b0 to fix it and also do a rebase.